### PR TITLE
man-db: add system account _man:_man

### PR DESCRIPTION
**Pull request by sgn**
man-db complains that:
> mandb: the setuid man user "man" does not exist

### CODE REVIEW

**Comment by Duncaen** [comment]
Hm can we use `_man` instead to avoid clashes with user accounts.

**Comment by zdtcd** [comment]
On 2019-10-31 07:14:21 -0700, Duncan Overbruck wrote:
> Hm can we use `_man` instead to avoid clashes with user accounts.

Let's me dig into man-db's source.

-- digging --

Looks like we can, updated patch is comming.

Anyway, man-db uses an extension of glibc's iconv.
Thus, it warns this on musl-libc:

> mandb: iconv_open ("UTF-8//IGNORE", "us-ascii"): Invalid argument

Is it worth to patch man-db ourself to remove `//IGNORE` on musl?

-- 
Danh

**Comment by zdtcd** [comment]
On 2019-11-01 07:43:49 +0700, Danh Doan wrote:
> Anyway, man-db uses an extension of glibc's iconv.
> Thus, it warns this on musl-libc:
> 
> > mandb: iconv_open ("UTF-8//IGNORE", "us-ascii"): Invalid argument
> 
> Is it worth to patch man-db ourself to remove `//IGNORE` on musl?

FWIW, this warning is printed out the man-db cronjob.
It'll be annoying to see syslog flooded with this useless message.

-- 
Danh

File: srcpkgs/man-db/template [hunk 1]
```diff
@@ -21,6 +22,9 @@ distfiles="${NONGNU_SITE}/${pkgname}/${pkgname}-${version}.tar.xz"
 checksum=5d4aacd9e8876d6a3203a889860c3524c293c38f04111a3350deab8a6cd3e261
 provides="man-0_1"
 lib32disabled=yes
+system_accounts="_man"
+man_homedir="/var/cache/man"
```
Comment (xtraeme) [inline srcpkgs/man-db/template]: should be `_man_xxx` instead

**Pull request merged**